aea/configurations/project.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the implementation of AEA agents project configuiration."""
+import os
+from shutil import rmtree
+from typing import Set
+
+from aea.cli.registry.fetch import fetch_agent
+from aea.cli.utils.context import Context
```

Comment:
Ok; on consideration, it seems to me the the project is actually part of the cli then...as configurations should not depend on cli

Reply:
yes, it requires refactoring on registry fetch, create helper for fetch and move it outside the cli

Reply:
agreed